packages/react-components/theme-designer/src/components/Sidebar/Form.tsx

```diff
@@ -159,35 +150,38 @@ export const Form: React.FC = () => {
 
   return (
     <div className={styles.root} role="tabpanel" aria-labelledby="Edit">
-      <Accordion defaultOpenItems={['1', '2', '3']} multiple>
-        {/* `multiple` allows for toggle of collapse as well as open multiple panels */}
+      <div className={styles.startHere}>
+        <Subtitle2Stronger as="h2">
+          Start Here
+        </Subtitle2Stronger>
+      </div>
+      <Accordion defaultOpenItems={['1']} collapsible>
         <AccordionItem value="1">
-          <AccordionHeader>
-            <Caption1Stronger>Step 1 - Color settings</Caption1Stronger>
-          </AccordionHeader>
-
+          <AccordionHeader size="large">Step 1 - Color settings</AccordionHeader>
           <AccordionPanel className={styles.accordionContainer}>
+            <Text as="h6">Use the Color Picker to select a color or enter a hex value in the text box below to generate a theme. Adjust the Hue Torsion and Vibrancy using the sliders below for further refinement.</Text>
             <div className={styles.inputs}>
               <div className={styles.labels}>
                 <Field label="Key color value">
                   <Input
-                    className={styles.keyColor}
-                    size="large"
-                    appearance="underline"
+                    appearance="outline"
                     value={keyColor}
                     onChange={handleKeyColorChange}
                     maxLength={7}
                     onBlur={handleKeyColorBlur}
-                  />
+                    style={{ width: '100px' }}
+                    />
+                  </Field>
+                  <Field label="Color Picker">
+                    <div className={styles.colorPicker} style={{ backgroundColor: keyColor }}>
```

Comment:
I think this would be simpler as a button rather than an input. Right now it's missing a ton of interactivity that'd you'd otherwise get for free from the component.